### PR TITLE
Add approximate member/presence count to guilds

### PR DESCRIPTION
**Pull request by JDJGInc**
approximate_member_count and approximate_presence_count, is now added.

## Checklist

- [x] If code changes were made then they have been tested
    - [x] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `black .`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

This should be better than the last one, and it works with black, I ran black . in the base folder what is now, will work with black.
All good :D

### CODE REVIEW

**Review by thesadru — APPROVED** [review]

**Review by Victorsitou — COMMENTED** [review]
Everything seems fine, just one thing, and sorry if I'm annoying with this, but can we follow the above syntax when adding new parameters in docstrings?

File: disnake/guild.py [hunk 1]
```diff
@@ -241,6 +241,17 @@ class Guild(Hashable):
...
+    approximate_member_count: Optional[:class:`int`]
+        The approximate number of members in the guild.
+
+        .. versionadded:: 2.0
+
+    approximate_presence_count: Optional[:class:`int`]
+        The approximate number of members currently active in the guild.
+        This includes idle, dnd, online, and invisible members. Offline members are excluded.
+
         .. versionadded:: 2.0
```
[the remark this answers is not in the document]
Comment (Hunter2807) [reply disnake/guild.py]: Don't u think this should be 2.3 since 2.2 is already released?

[the remark this answers is not in the document]
Comment (Victorsitou) [reply disnake/guild.py]: I don't know why this has been posted twice, I requested these changes last night :/

**Comment by JDJGInc** [comment]
It should be fine, the new doc string follows the same format as the old one. Black also has no issues with this

**Comment by Victorsitou** [comment]
> It should be fine, the new doc string follows the same format as the old one. Black also has no issues with this. If you meant that it was the wrong version, that I can seem but if there is an issue with the doc string I will gladly fix it

What I meant is to delete those extra spaces
image

**Comment by JDJGInc** [comment]
Can you check now?
I don't have the necessary browser to do the exam, does this work though?

**Comment by Victorsitou** [comment]
> Can you check now?

All good now, thank you!

**Review by Victorsitou — APPROVED** [review]

**Review by shiftinv — APPROVED** [review]

File: disnake/guild.py [hunk 2]
```diff
@@ -286,6 +304,8 @@ class Guild(Hashable):
         "_stage_instances",
         "_threads",
         "_application_commands",
+        "approximate_member_count",
+        "approximate_presence_count",
```
Comment (jgayfer) [inline disnake/guild.py:308]: Maybe we keep this alphabetical?

**Review by jgayfer — COMMENTED** [review]
Are there any implications to the request taking longer? I assume there's a reason this is optional.

File: disnake/guild.py (hunk 2 above)
Comment (Victorsitou) [reply disnake/guild.py:308]: The list doesn't seem to be alphabetical, but I suggest adding those two lines below the "nsfw" one.

Comment (JDJGInc) [reply disnake/guild.py:308]: it's underneath the other ones, since it's new.

**Review by thesadru — APPROVED** [review]

**Pull request merged**